<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Integration time [s]</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtIntTime">
        <property name="text">
         <string>0.5</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Step angle [°]</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtStpAngle">
        <property name="text">
         <string>0.1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Initial angle [°]</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="txtIniAngle">
        <property name="text">
         <string>-10</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Final angle [°]</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="txtFinAngle">
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Set rotatore lamina1</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Step angle[°]</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="txtStpAngleLamina">
        <property name="text">
         <string>0.05</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Initial angle[°]</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="txtIniAngleLamina">
        <property name="text">
         <string>-5</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Final angle[°]</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="txtFinAngleLamina">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Reflection angle</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="txtZeroAngle">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Total time [min]</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QLineEdit" name="txtTotTime">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Time left [min]</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QLineEdit" name="txtTimeLeft">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="19" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>TTBuffer number</string>
        </property>
       </widget>
      </item>
      <item row="19" column="1">
       <widget class="QLineEdit" name="txtBufferNo">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="21" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>(glass) Rotator serial number</string>
        </property>
       </widget>
      </item>
      <item row="21" column="1">
       <widget class="QLineEdit" name="txtSN">
        <property name="text">
         <string>83825706</string>
        </property>
       </widget>
      </item>
      <item row="22" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Orthogonal angle</string>
        </property>
       </widget>
      </item>
      <item row="22" column="1">
       <widget class="QLineEdit" name="txtAngleGlass">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="25" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3"/>
      </item>
      <item row="26" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>(Wplate1) Rotator serial number</string>
        </property>
       </widget>
      </item>
      <item row="26" column="1">
       <widget class="QLineEdit" name="txtSNlamina">
        <property name="text">
         <string>83825836</string>
        </property>
       </widget>
      </item>
      <item row="27" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Orthogonal angle</string>
        </property>
       </widget>
      </item>
      <item row="27" column="1">
       <widget class="QLineEdit" name="txtAngleWplate1">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="29" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>(Wplate2) Rotator serial number</string>
        </property>
       </widget>
      </item>
      <item row="29" column="1">
       <widget class="QLineEdit" name="txtSNlamina2">
        <property name="text">
         <string>83865112</string>
        </property>
       </widget>
      </item>
      <item row="30" column="0">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Orthogonal angle</string>
        </property>
       </widget>
      </item>
      <item row="30" column="1">
       <widget class="QLineEdit" name="txtAngleWplate2">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="31" column="1">
       <widget class="QPushButton" name="btnConnect">
        <property name="text">
         <string>Connect Rotator</string>
        </property>
       </widget>
      </item>
      <item row="32" column="1">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="33" column="0">
       <widget class="QPushButton" name="btnSaveData">
        <property name="text">
         <string>Save Data</string>
        </property>
       </widget>
      </item>
      <item row="33" column="1">
       <widget class="QLineEdit" name="txtFileName"/>
      </item>
      <item row="34" column="1">
       <widget class="QPushButton" name="btnStart">
        <property name="text">
         <string>Start Acquisition</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="PlotWidget" name="pltVisibility"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="1">
         <widget class="QLineEdit" name="txtMaxVis0">
          <property name="text">
           <string>0%</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="txtMaxVis1">
          <property name="text">
           <string>0%</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Max visibility channel 1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Max visibility channel 0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
